src/V3Timing.cpp

```diff
@@ -50,6 +50,7 @@
 #include "V3Const.h"
 #include "V3EmitV.h"
 #include "V3Graph.h"
+#include "V3SenExprBuilder.h"
```

Comment:
This should be in a later patch I believe.

Reply:
Correct, fixed.